src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -418,6 +419,7 @@ pub struct CreateSinkStatement<T: AstInfo> {
     pub with_options: Vec<SqlOption<T>>,
```

Comment:
Looks like the BNF diagram never got the `with_options` field added to it; would you mind including it in this PR?